redaxo/src/core/lib/sql/sql.php

```diff
@@ -148,7 +148,7 @@ protected function selectDB($db)
             if ('cli' === PHP_SAPI) {
                 throw new rex_sql_could_not_connect_exception("Could not connect to database.\n\nConsider starting either the web-based or console-based REDAXO setup to configure the database connection settings.", $e, $this);
             }
-            throw new rex_sql_could_not_connect_exception('Could not connect to database', $e, $this);
+            throw new rex_sql_could_not_connect_exception('Could not connect to database (DB: ' . $db . ')', $e, $this);
```

Comment:
In Zeile 149 am besten auch noch ergänzen.

Reply:
Habe ich ergänzt.